images/linux/Ubuntu1804-README.md

```diff
@@ -107,6 +107,7 @@
 - Hub CLI 2.14.2
 - Netlify CLI 6.0.9
 - OpenShift CLI 4.8.4
+- Oracle Cloud Infrastructure CLI 3.0.0
```

Comment:
You shouldn't modify READMEs as they are generated automatically.

Reply:
Whoops. Let me revert that.